stdlib/os/__init__.pyi

```diff
@@ -173,14 +180,14 @@ if sys.platform != "win32" and sys.platform != "darwin":
     ST_NODEV: int
     ST_NODIRATIME: int
     ST_NOEXEC: int
-    ST_NOSUID: int
-    ST_RDONLY: int
     ST_RELATIME: int
     ST_SYNCHRONOUS: int
     ST_WRITE: int
 
 if sys.platform != "win32":
     NGROUPS_MAX: int
+    ST_NOSUID: int
+    ST_RDONLY: int
```

Comment:
This change has no corresponding change in `posix.pyi`.

Reply:
Thanks

Reply:
Maybe we should add a check for that in `check_consistent.py`?

Reply:
That might be more trouble than it's worth, though, I suppose.

Reply:
stubtest would help, but `py3_common.txt` excludes `(posix.ST_[A-Z]+)?`.